### PR TITLE
add team member

### PR DESCRIPTION
**Pull request by tejaswinichile**
#314

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@breezyboa _a

File: app/javascript/Admin/Main.elm [hunk 1]
```diff
@@ -762,6 +770,26 @@ update msg model =
                 , Cmd.map TeamListMsg teamListCmd
                 )
 
+        TeamCreateMsg tcmsg ->
+            let
+                currentPageModel =
+                    case model.currentPage of
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: use getPage function to get past PageStatus type

**Comment by tejaswinichile** [comment]
@breezyboa _a Please check

**Comment by breezyboa** [comment]
Elm code looks good
@abdul-shajin _a can you review rails code

**Comment by abdul-shajin** [comment]
Looks good to me @tejaswinichile  _a

**Pull request merged**